R/REST_helpers.R

```diff
@@ -36,12 +36,12 @@ sbtools_POST <- function(url, body, session, timeout = default_timeout(), ...){
 #' @keywords internal
 sbtools_GET <- function(url, ..., session, timeout = default_timeout()){
 	
-	supported_types <- c('text/plain','application/json')
+	supported_types <- c('text/plain','application/json','application/x-gzip')
```

Comment:
@aappling-usgs how to you feel about changing this to unsupported types to make life easier on us?